packages/grafana-data/src/utils/dataLinks.ts

```diff
@@ -12,3 +17,80 @@ export const DataLinkBuiltInVars = {
   // name of the calculation represented by the value
   valueCalc: '__value.calc',
 };
+
+export function mapInternalLinkToExplore(
```

Comment:
I had to move this inside the grafana/data because otherwise there would be cyclic imports from the field overrides code.